lms/static/sass/base/_layouts.scss

```diff
@@ -3,7 +3,8 @@
 
 // overriding existing styles on the body element
 // .view-incourse scopes these rules to be specific to student being in a course
-body.view-incourse {
+body.view-incourse,
+body.is-in-course {
```

Comment:
What pages are .is-in-course being applied to? It isn't clear to me here at a glance (and a quick edx-platform/master search didn't yield any matches). Is this being applied when view-teams is applied?